packages/runtime-common/realm.ts

```diff
@@ -839,6 +841,7 @@ export class Realm {
     requestContext: RequestContext,
   ): Promise<ResponseWithNodeStream> {
     let headers = {
+      'x-created': formatRFC7231(ref.created * 1000),
```

Comment:
I didn't find a "natural" header for file creation date (similarly to `Last-Modified`), so I added a custom one with a `x-*` name